index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+    </style>
+    <title>portfolio</title>
+  </head>
+  <body>
+    <div class="flex-container">
+      <nav class="nav">
+        <h3 id="logo">My Logo</h3>
+        <img src="images/Union.png" alt="navbar" />
+      </nav>
+      <header>
```

Comment:
Please ensure that you remove the `header` tag from this section.

This sections should be like this:
```
<section>
            <h2 id="intro">I'm Abdul Hadi Jafari Glad to see you!</h2>
            <p id="bio">
              I’m a software developer! I can help you build a product , feature
              or website Look through some of my work and experience! If you like
              what you see and have a project you need coded, don’t hestiate to
              contact me.
            </p>
         <div class="social">
          <h3 id="in-touch">LET'S CONNOCT</h3>
          <ul class="social-icons">
            <li>
              <img src="images/twitter.svg" alt="twitter" />
            </li>
            <li>
              <img src="images/Linkedin.svg" alt="Linkedin"/>
            </li>
            <li>
              <img src="images/medium.svg" alt="medium"/>
            </li>
            <li>
              <img src="images/github.svg" alt="github" />
            </li>
            <li>
              <img src="images/angellist.svg" alt="angellist" />
            </li>
          </ul>
        </div>
</section>
```